working-groups/README.md

```diff
@@ -1,36 +1,87 @@
+- **Proposal Requirements**  
+  Each proposal must include:
+  1. Name / Title  
+  2. Scope (what the group will address)  
+  3. Objectives & Deliverables  
+  4. Initial Membership (lead(s) and core participants)  
+  5. Timeline or milestones (if applicable)  
 
-## Governance Rules for Working Groups
+- **Submission**  
+  Proposals are filed as GitHub issues in the [`fairpm/tsc`](https://github.com/fairpm/tsc) repository with the label `wg-proposal`.
 
-The following is a temporary set of governance rules for each Working Group. It is the intention that under a future TSC Charter, working groups shall have the ability to self-govern.
+## Review & Approval
 
-Each Working Group should copy the [`sample-readme.md`](https://github.com/fairpm/tsc/tree/main/working-groups/sample-readme.md) file as a template and place it within their designated subfolder under `/working-groups/` in the this repository (e.g., `/working-groups/community/README.md`). This file serves as the project “front door” for the Working Group and should be customized with the relevant details in `{}`. If the Working Group operates from a separate repository, the file should instead be placed in that repository’s main README or equivalent homepage.
+- **Evaluation Criteria**  
+  - Relevance to FAIR project goals  
+  - Clarity of objectives  
+  - Feasibility (time, scope, resources)  
+  - Membership and leadership
 
-Working Groups have a high degree of flexibility in how they work, but the following should remain consistent across all groups:
+- **Decision Process**  
+  - The TSC reviews the proposal in a meeting or asynchronously via GitHub.  
+  - A simple majority vote is required for approval.  
+  - Decision and rationale are documented in the proposal issue. 
 
-* All TSC policies, including the Code of Conduct and Licensing policies must be included by reference;
-* Workspaces for all Working Groups, such as Slack channels, forums, mailing lists or other spaces must be posted centrally for easy access;
-* All Working Groups must share details on how someone can contribute to them;
+
+## Operations & Tracking
+
+- **Dedicated Repository**  
+  Approved WGs will normally receive their own repository under the FAIR GitHub organization, named `wg-<name>` (e.g., `wg-docs`).  
+
+- **Tracking Requirements**  
+  - Agendas, meeting notes, status reports, and deliverables stored in the WG repo. 
+  - Work items tracked via GitHub issues and project boards.  
+
+- **Communication**  
+  Optional Slack channels or mailing lists may be created, but GitHub is the system of record.
+
+
+## Review & Sunset
+
+- **Periodic Review**  
+  Every 6 months, each WG must review its purpose, progress, and continued relevance.  
+
+- **Dormancy**  
+  If a WG is inactive for 3 months, it will be marked *Dormant*.  
```

Comment:
I know this is in the weeds, but just curious how other projects do this. For example, renaming a WG repository or their corresponding folder in the TSC docs to something like `wg-xyz-DORMANT`?